ENGAGEHF/Home.swift

```diff
@@ -43,6 +47,21 @@ struct HomeView: View {
                 .tabItem {
                     Label("Home", systemImage: "house")
                 }
+            HeartHealth(presentingAccount: $presentingAccount)
+                .tag(Tabs.heart)
+                .tabItem {
+                    Label("Heart Health", systemImage: "heart")
+                }
+            Medications(presentingAccount: $presentingAccount)
+                .tag(Tabs.medications)
+                .tabItem {
+                    Label("Medications", systemImage: "pill")
+                }
+            Education(presentingAccount: $presentingAccount)
+                .tag(Tabs.education)
+                .tabItem {
+                    Label("Education", systemImage: "brain")
+                }
```

Comment:
Wondering if we should already display them if they don't have any content as this point. We might add them as we add different content (e.g. a contacts section in the education tab and then include this in this PR).